src/update-feeds/update-feeds.ts

```diff
@@ -171,7 +169,7 @@ export const processBatch = async (batch: ReadDapiWithIndexResponse[], chainId:
         dapi.decodedDataFeed.beacons.flatMap((dataFeed) => `${url}/${dataFeed.airnodeAddress}`)
       );
 
-      draft.signedApiUrlStore = receivedUrls.flat();
+      draft.signedApiUrlStore[providerName] = receivedUrls.flat();
```

Comment:
We actually need to index by chainId as well, because it's possible to have two "infura" providers on different chains.

Reply:
Done :D 

It is keyed as `signedApiUrlStore: Record<ChainId, Record<Provider, string[]>>` - which has to be unique.